contracts-periphery/script/Deploy.s.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
```

Comment:
scopelint currently errors if the `script` dir is missing, so this is just a placeholder script until we write the actual deploy scripts